SKIP-REQUIRE-JAVADOC

```diff
@@ -0,0 +1 @@
+Delete after merge.
```

Comment:
Why is this file necessary?   I don't see wide-ranging uninteresting edits in this pull request.

Reply:
The problem is that NodeVisitor has 80 undocumented methods.  I would have to document them all to get the Javadoc test to pass. See https://github.com/typetools/checker-framework/runs/4509004521.